x-pack/plugins/cases/public/containers/use_get_cases.tsx

```diff
@@ -230,6 +231,15 @@ export const useGetCases = (
           dispatch({ type: 'FETCH_UPDATE_CASE_SUCCESS' });
           fetchCases(state.filterOptions, state.queryParams);
           refetchCasesStatus();
+          if (caseData) {
+            toasts.addSuccess({
+              title: i18n.UPDATED_CASE(caseData.title),
+              text:
+                updateKey === 'status' && caseData.totalAlerts > 0
+                  ? i18n.STATUS_CHANGED_TOASTER_TEXT
```

Comment:
I know you didn't create the text but it currently reads:

`'Alerts in this case have been also had their status updated'`

Can we change that to `'Alerts in this case have also had their status updated'` or maybe: `The status of attached alerts in this case were also updated`

@nastasha-solomon @szabosteve what do you think?

Reply:
Hm, how about:
Updated the statuses of attached alerts.

Reply:
@nastasha-solomon great. I'll do the change!